assets/airports/cyhz.json

```diff
@@ -588,96 +588,96 @@
-        "IGVOP": ["N45d03.52", "W063d38.66"],
-        "ILOTA": ["N44d52.52", "W063d48.90"],
-        "IMANO": ["N44d55.98", "W063d35.78"],
-        "IMESA": ["N44d56.14", "W063d39.47"],
-        "KEBNI": ["N45d07.89", "W064d11.37"],
-        "KEBOT": ["N44d59.18", "W063d22.63"],
-        "KESMO": ["N44d54.01", "W063d20.99"],
-        "KIXIT": ["N44d48.33", "W063d08.31"],
-        "LEROS": ["N45d02.20", "W063d21.47"],
-        "LIRLA": ["N45d01.25", "W062d50.13"],
-        "LOGPO": ["N44d58.98", "W063d34.13"],
-        "LOPMA": ["N44d48.59", "W063d34.93"],
-        "MENVO": ["N44d53.86", "W063d11.68"],
-        "MITIK": ["N44d56.89", "W063d26.74"],
-        "MIVON": ["N44d59.76", "W063d32.50"],
-        "MIXIV": ["N45d02.10", "W063d35.80"],
-        "MODIR": ["N44d55.53", "W063d47.22"],
-        "MOWND": ["N45d22.60", "W066d39.43"],
-        "MUSVI": ["N44d58.52", "W063d25.12"],
-        "MUVOX": ["N44d51.85", "W064d12.20"],
-        "NOTOP": ["N45d27.16", "W062d00.71"],
-        "NOTUB": ["N44d40.87", "W063d28.03"],
-        "NUBUK": ["N44d57.85", "W063d36.30"],
-        "NUTBY": ["N45d41.25", "W063d14.80"],
-        "OBNOM": ["N44d40.81", "W063d34.02"],
-        "ODKAS": ["N44d43.68", "W063d39.76"],
-        "OTSUD": ["N44d46.50", "W063d29.03"],
-        "OTVUV": ["N44d59.32", "W063d15.71"],
-        "PAULO": ["N45d15.18", "W063d20.13"],
-        "PENLU": ["N44d50.61", "W063d25.02"],
-        "PEPTA": ["N44d47.60", "W063d27.33"],
-        "PESNU": ["N44d58.79", "W063d26.16"],
-        "RONPO": ["N44d40.34", "W063d51.58"],
-        "RUDBU": ["N44d51.87", "W063d23.11"],
-        "TAKIM": ["N44d52.47", "W063d28.74"],
-        "TETAR": ["N44d59.42", "W063d42.71"],
-        "TIDON": ["N44d59.78", "W063d31.13"],
-        "TILAD": ["N44d57.92", "W063d24.41"],
-        "TUBEG": ["N45d00.08", "W063d31.74"],
-        "TUSKY": ["N43d33.90", "W067d00.00"],
-        "UKVAT": ["N44d53.40", "W063d19.61"],
-        "URMIN": ["N45d00.13", "W063d47.11"],
-        "VIDGA": ["N44d50.40", "W063d21.38"],
-        "VIMLA": ["N44d58.76", "W063d29.09"],
-        "VITAV": ["N44d57.49", "W063d38.81"],
-        "VOKIL": ["N44d47.15", "W063d18.13"],
-        "XIBUL": ["N45d17.70", "W066d39m43"],
-        "XOMRO": ["N44d42.93", "W063d40.38"],
-        "XOVMU": ["N44d55.98", "W063d35.76"],
-        "XUBKO": ["N45d03.84", "W063d19.77"],
-        "XUGID": ["N44d54.04", "W063d31.89"],
-        "YHZ"  : ["N44d55.39", "W063d24.11"],
-        "YQI"  : ["N43d49.51", "W066d04.95"],
-        "YQM"  : ["N46d11.34", "W064d34.25"],
-        "YZX"  : ["N44d55.37", "W065d06.11"],
-        "ZHZ"  : ["N44d48.14", "W063d35.38"],
-        "ZNS"  : ["N44d58.00", "W063d25.62"]
-    },
+    "fixes": [
+        {"name": "AGMIR", "coordinates": ["N45d13.70", "W061d57.33"]},
+        {"name": "ALLEX", "coordinates": ["N44d25.00", "W067d00.00"]},
+        {"name": "AMUNA", "coordinates": ["N44d57.74", "W063d07.54"]},
+        {"name": "ARDEE", "coordinates": ["N45d07.40", "W064d23.17"]},
```

Comment:
This pattern worries me. If using an array of objects rather than the fix name as the key of an object property, we introduce the possibility of duplicated fixes. As it stands currently, this is protected by a lint error (attempting to define two of the same key in an object). I would suggest instead something like:
```
"fixes": {
        "AGMIR": ["N45d13.70", "W061d57.33"],
        "ALLEX": ["N44d25.00", "W067d00.00"],
        "AMUNA": {"coordinates": ["N44d57.74", "W063d07.54"], "spoken": "amoonuh"},
        "ARDEE", {"coordinates": ["N45d07.40", "W064d23.17"], "spoken": "party},
}
```

and then from the FixModel constructor, we can simply check if `Array.isArray(fixData)`-- if so, treat as positional array with spoken undefined, and else `const { coordinates, spoken } = fixData;` to pull out the two values and use them. If the fix data provided is not an object with those two properties, an error will be thrown naturally, ensuring the existence of the parameters without any other manual defensiveness. 😄

Reply:
Wilco! That sounds great.